### ISSUE TITLE
Cannot find module 'app/secrets/secrets.json'

### ISSUE THREAD

**Issue by mubaris** [ghostwriternr/lowpolify#1]
Output of `node server.js`

```
module.js:472
    throw err;
    ^

Error: Cannot find module '~/dev/lowpolify/app/secrets/secrets.json'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/zeus/dev/lowpolify/app/routes/clarifai-routes.js:8:16)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
```

**Comment by pawndev** [comment]
Same

**Comment by oleksandr-shvets** [comment]
This is clarifai api credentials. You can get it on https://www.clarifai.com
```
{
    "clientId": "",
    "clientSecret": "",
    "access": {
        "access_token": ""
    }
}
```

**Comment by ghostwriternr** [comment]
(Ouch. I must have missed the notifications for the previous comments.)
secrets.json should be structured exactly like how @alexander-shvets has mentioned (Thank you!)
The final .json would look something like this:
```json
{
    "clientId": "",
    "clientSecret": "",
    "access": {
        "access_token": "",
        "token_type": "Bearer",
        "expires_in": 176400,
        "scope": "api_access_write api_access api_access_read"
    }
}
```

The code will automatically fill in the `access` key. You'd just have to add your `clientId` and `clientSecret`.
That being said, I just noticed today [on their site](https://www.clarifai.com/developer/guide/authentication#authentication) that they're stopping support for Access Tokens/Client ID & Secret.

It is a separate issue, but I'll try to update the code as and when I get the time. I'd absolutely love a PR as well though, in case you fix it 😄 [ EDIT: Added issue #2 ]

**Comment by ghostwriternr** [comment]
Following the changes introduced in ff6f7dd, the new structure of `secrets.json` would be:
```json
{
  "apiKey": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
}
```
The link for creating your own developer keys is https://clarifai.com/developer/account/keys.

I have updated the instructions in the README as well. Thanks a lot for pointing this out!

Closing this issue now.
